instrumentation/jmx-metrics/library/src/main/resources/jmx/rules/jvm.yaml

```
@@ -96,6 +96,12 @@ rules:
         type: gauge
         unit: '1'
         desc: Recent CPU utilization for the whole system as reported by the JVM.
+      # jvm.file_descriptor.count (experimental)
+      OpenFileDescriptorCount:
+        metric: file_descriptor.count
+        type: updowncounter
+        unit: "{file_descriptors}"
```

```suggestion
        unit: "{file_descriptor}"
```

Units should not use plural form https://opentelemetry.io/docs/specs/semconv/general/metrics/#instrument-units